workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -0,0 +1,685 @@
+                            "$comment": "{\"term\": \"decimals\", \"@id\": \"https://schema.org/Number\"}",
+                            "title": "decimals",
+                            "description": "Allowed number of decimals for the listed token.",
+                            "type": "integer",
+                            "examples": [18]
+                        },
+                        "tokenLogoURI": {
+                            "$comment": "{\"term\": \"tokenLogoURI\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "tokenLogoURI",
+                            "description": "URI or URL of the token logo.",
+                            "type": "string"
+                            "examples": ["https://polygonscan.com/token/images/matic_32.png"]
+                        },
+                        "createdAt": {
+                            "$comment": "{\"term\": \"createdAt\", \"@id\": \"https://schema.org/datePublished\"}",
+                            "title": "createdAt",
+                            "description": "Date and time token was created",
+                            "type": "string",
+                            "examples": ["2020-05-31"]
+                        },
+                        "updatedAt": {
+                            "$comment": "{\"term\": \"updatedAt\", \"@id\": \"https://schema.org/dateModified\"}",
+                            "title": "updatedAt",
+                            "description": "Date and time token was updated",
+                            "type": "string"
+                            "examples": ["2020-05-31"]
+                        },
+                        "extensions": {
+                            "title": "extensions",
+                            "description": "Extension to the token list entry to specify an origin chain if the token entry refers to another chain other than the origin chain of the token",
+                            "type": "array",
+                            "items": {
+                                "type":"object",
+                                "required": [
+                                    "rootChainId",
+                                    "rootChainURI",
+                                    "rootAddress",
+                                ],
+                                "properties": {
+                                    "rootChainId": {
+                                        "$comment": "{\"term\": \"rootChainId\", \"@id\": \"https://schema.org/identifier\"}",
+                                        "title": "rootChainId",
+                                        "description": "The typically used number identifier for the root chain on which the token was originally issued.",
+                                        "type": "number",
+                                        "examples": [137]
+                                    },
+                                    "rootChainURI": {
```

Comment:
Is this the genesis of a fork or the genesis of the chain. Example: Ethereum classic & Ethereum have the same genesis block and resolve to the same blocks prior to block 192k

Reply:
@realcodywburns the disambiguation parameters are `chainId` and `chainURI` in this instance.